translator/tocwconfig/sampleConfig/emf_and_kubernetes_config.yaml

```diff
@@ -34,7 +34,7 @@ exporters:
     disable_metric_extraction: true
     eks_fargate_container_insights_enabled: false
     certificate_file_path: ""
-    endpoint: ""
+    endpoint: "https://fake_endpoint"
```

Comment:
Just to confirm, were these supposed to be overridden before, but we weren't doing translation correctly in the EMF exporter for it to be reflected? Or is this new behavior?

Reply:
Now we are using the yaml translator. Previously we used toml translator. I think thats the reason it might be the erroring out as mismatch.